mole/models/path.py

```diff
@@ -1,38 +1,37 @@
 from __future__ import annotations
 from mole.core.data import Path
 from mole.grouping import get_grouper
+from mole.models import IndexedLabeledEnum
+from mole.services.ai import AiVulnerabilityReport
 from typing import Dict, List, Optional, Tuple
```

Comment:
This is more a generic note to both of us, since we indeed require at least python 3.10 we can start using native builtin like `list`, `set`, `dict`, `X | None`, ...